### DISCUSSION TITLE
Custom emojis support

### DISCUSSION THREAD

**Discussion by hatkidchan** [hyperspacedev/hyperspace#254] [Ideas]
It'd be nice to show custom emojis when you compose posts, like Mastodon web does:
(yes, I know there is "Insert emoji" button with "Custom" tab in it, it's just not as convenient to use as Mastodon default one)

![image](https://user-images.githubusercontent.com/35804000/172586028-f2b927a0-ae13-4ee1-9c67-5a4078925de0.png)
![image](https://user-images.githubusercontent.com/35804000/172585965-597a8fcb-9e85-4eb2-a820-54d53278e2a3.png)

Also custom emojis don't show up in "Notifications" tab:
![image](https://user-images.githubusercontent.com/35804000/172586244-89b9b1cb-ab32-4eb7-a026-fa79d3be7053.png)
![image](https://user-images.githubusercontent.com/35804000/172586326-566ab7f3-a4b5-4451-af53-1b30c0d1ea4a.png)

**Comment by alicerunsonfedora** [comment]
We'll keep this in mind for Starlight and Luna, provided that we can implement it in the tools we use to build those.
